Src/AutoFixture/RandomDateTimeOffsetSequenceGenerator.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using Ploeh.AutoFixture.Kernel;
+
+namespace Ploeh.AutoFixture
+{
+    /// <summary>
+    /// Creates random <see cref="DateTimeOffset"/> specimens.
+    /// </summary>
+    /// <remarks>
+    /// The generated <see cref="DateTimeOffset"/> UTC values will be within
+    /// a range of ± two years from today's date,
+    /// unless a different range has been specified in the constructor.
+    /// </remarks>
+    public class RandomDateTimeOffsetSequenceGenerator : ISpecimenBuilder
+    {
+        private readonly RandomNumericSequenceGenerator _dateRandomizer;
```

Comment:
From my understanding I see 2 ways to re-use the existing generator:
1. Simply use the suggested `RandomDateTimeSequenceGenerator` instead of `RandomNumericSequenceGenerator ` and manual work with offsets when you calculate the number of ticks.

2. Don't manage the `DateTime` creation at all and use the `specimenContext.Resolve(typeof(DateTime))` instead. This way you will not care about `DateTime` creation so it could be configured/customized separately. Later you could create `DateTimeOffset` based on that `DateTime` value.

I'd prefer the second way as it's more natural for this project and is more customizable.
Do you see any issues with such approach?